examples/with-vitest/package.json

```diff
@@ -13,14 +13,11 @@
     "server-only": "^0.0.1"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "6.1.2",
     "@testing-library/react": "14.0.0",
-    "@testing-library/user-event": "14.4.3",
     "@types/node": "20.5.9",
     "@types/react": "18.2.8",
-    "@types/testing-library__jest-dom": "6.0.0",
-    "@vitejs/plugin-react": "4.0.4",
-    "jsdom": "^22.1.0",
+    "@vitejs/plugin-react": "^4.2.1",
+    "jsdom": "^23.0.1",
     "typescript": "5.2.2",
```

Comment:
We might also consider updating these to all latest versions too (follow up PR?)

Reply:
sgtm!